qa/suites/rados/mgr/tasks/orchestrator_cli.yaml

```diff
@@ -8,8 +8,8 @@ tasks:
       log-whitelist:
         - overall HEALTH_
         - \(MGR_DOWN\)
-        - \(MGR_INSIGHTS_WARNING\)
-        - \(insights_health_check
```

Comment:
are these lines meant to be dropped?

Reply:
Yep, we never should see insights warnings in our log file, right?

Reply:
Ah, right